README.md

```diff
@@ -58,6 +58,10 @@ For a local PostgreSQL instance with a username and password one could call:
 ```
 TEST_ONLINE=postgresql://postgres:postgres@localhost:5432/postgres prove -l t/*.t t/*.t.js
 ```
+### Local Development
+
+An quick way to build and run the dashboard and its dependencies can be with docker.
+More info here https://gitlab.suse.de/qsf-u/dev-dashboard
```

Comment:
"An quick way … can be with docker" is really a weird way to phrase this. The article and casing is also wrong. I'd also avoid mentioning this as first and only suggestion for local development.

```suggestion
### Further notes
A containerized environment could be used to build and run the dashboard and its dependencies.
For a concrete example, checkout the (so far) internal documentation under
https://gitlab.suse.de/qsf-u/dev-dashboard.
```